### ISSUE TITLE
Sign out button loses active state when rerendered

### ISSUE THREAD

**Issue by djrenren** [chezbob/chezbob#17]
If you've noticed the signout button acting a little weird. That's because it has! If the 1s rerender timer goes off while you're between pointer-down and pointer-up, then the "pressed" state is lost and your click won't count.

This is happening because we're doing:

```js
header.innerHTML = `...<button ...>${time_remaining}</button>`;
```

Because the contents of the button are changing, a new button element is created and replaces the one we're currently pressing.
To get around this, we can just use a custom element called `<session-timer>` that displays the time remaining using the Shadow DOM. The details are pretty simple even if that sounds intimidating.

Basically, we'll always render `<button><session-timer></session-timer></button>` and `session-timer` will internally handle displaying the time.

**Comment by zeldon-dev** [comment]
yo djrenren can you follow me back
